doc/dev/peering.rst

```diff
@@ -45,39 +45,36 @@ Concepts
    writes to objects in a placement group.
 
 *replica*
-   a non-primary OSD in the *acting set* for a placement group
-   (and who has been recognized as such and *activated* by the primary).
+   a non-primary OSD in the *acting set* of a placement group. A replica has
+   been recognized as a non-primary OSD and has been *activated* by the
+   primary.
 
 *stray*
-   an OSD who is not a member of the current *acting set*, but
-   has not yet been told that it can delete its copies of a
-   particular placement group.
+   an OSD that is not a member of the current *acting set* and has not yet been
+   told that it can delete its copies of a particular placement group.
```

Comment:
s/that it can/to/ ?

Reply:
I like that better.

Reply:
"can' makes it sound like the OSD has a choice in the matter.